Tests/FeatherWidgets.TestIntegration/Navigation/LanguageSelectorWidgetTests.cs

```diff
@@ -8,15 +8,19 @@
 using MbUnit.Framework;
 using Telerik.Sitefinity.Abstractions;
 using Telerik.Sitefinity.Frontend.Navigation.Mvc.Controllers;
+using Telerik.Sitefinity.Frontend.Navigation.Mvc.Models.LanguageSelector;
+using Telerik.Sitefinity.Frontend.News.Mvc.Controllers;
 using Telerik.Sitefinity.Frontend.TestUtilities;
 using Telerik.Sitefinity.Localization;
 using Telerik.Sitefinity.Modules.Pages;
 using Telerik.Sitefinity.Mvc.Proxy;
+using Telerik.Sitefinity.News.Model;
 using Telerik.Sitefinity.Pages.Model;
 using Telerik.Sitefinity.TestIntegration.Data.Content;
 using Telerik.Sitefinity.Utilities.HtmlParsing;
 using Telerik.Sitefinity.Web;
 using Telerik.WebTestRunner.Server.Attributes;
+using NewsOperationsContext = Telerik.Sitefinity.TestUtilities.CommonOperations.NewsOperations;
```

Comment:
This kind of using alias is used when there is another NewsOperations from different namespace. Just add using for Telerik.Sitefinity.TestUtilities.CommonOperations and make sure that classes from this namespace are used without the namespace in the type name.